<!DOCTYPE html>
<html>
<head>
  <title>Mats Nilsson Lab</title>
  <meta charset="utf-8"/>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link href='https://fonts.googleapis.com/css?family=Roboto:300,400,700' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" type="text/css" href="./css/main.css">

</head>
<body>
  <header class="container">
	  <link rel="icon" href="img/moldia.png">

    <div class="navigation">
      <ul>
        <li><a class="page-link" href=""><font color=#ccd5df>Home</font></a></a></li>
		<li><a class="page-link" href="research/"><font color=#ccd5df>Research</font></a></li>
        <li><a class="page-link" href="publications/"><font color=#ccd5df>Publications</font></a></li>
	<li><a class="page-link" href="facility/"><font color=#ccd5df>ISS Facility</font></a></li>
        <li><a class="page-link" href="people/"><font color=#ccd5df>People</font></a></li>
        <li><a class="page-link" href="links/"><font color=#ccd5df>Links</font></a></li>
      </ul>
    </div>
  </header>
  
	<p>&nbsp;</p>
	
	
<!-- LARGE TITLE BANNER -->	
	
  <section class="jumbotron">
    <div class="container">
      <div class="row text-center">
	      <h1>Mats Nilsson Lab</h1>
	      <br><br><br> 
	<img src="img/logo1.png" style="width:500px">
      </div>
    </div>
  </section>
  
  <p>&nbsp;</p>
	
<!-- News post section -->	
  
  <div class="wrap">
    <div class="post">
      <div class="news">
        
<!-- Pinned headline -->
	      
<!--	
	<h2>Title</h2>
	  <ul class="posts">
            <li>
              <span class="post-link"><p><a href="___"</a></p></span>
	    </li>
	  </ul>
	      
        <p>&nbsp;</p>
-->	      
	    
<!-- End pinned headline     
	      
        <h2>Lab News:</h2>
        <p>&nbsp;</p>
        <ul class="posts">
          -->
<!-- TO ADD NEWS ITEM, COPY AND PASTE TEMPLATE BELOW:	  

          <li>
            <span class="post-date">TYPE DATE HERE</span>
            <span class="post-link">POST TITLE HERE</span>
          </li>
          <div class="well">
            <li class="media">
              <div class="media-body">
                <img class="pull-left" style="margin-right:10px;" src="IMAGE FILE HERE" height="300px";>
                  <strong class="media-heading"><a href="LINK TO ARTICLE/NEWS HERE">TITLE FOR LINK HERE</a></strong>
                <p><small>SHORTHAND AUTHORS HERE</small></p>
                 <p>PARAGRAPH TEXT HERE</p>
                FOR ADDITIONAL LINKS INCLUDE LINES BELOW, OTHERWISE REMOVE
                <p><a href="LINK HERE">TITLE FOR LINK</a></p>
                <p><a href="LINK HERE">TITLE FOR LINK</a></p>
              </div>
             </li>
           </div>
           <p>&nbsp;</p>

:END TEMPLATE -->
   
<div class="row">
	    <div class=" col-sm-7  col-md-7 col-lg-7" align="left">  
              <h4>About the lab</h4>
		    <p>Our work is focused on development of novel molecular analysis concept for use in research and diagnostics, 
		    with primary focus on infectious and cancer diagnostics. We address development of both fundamental assay 
		    architecture and novel devices. Our research is based on a cross-disciplinary approach involving extensive 
		    collaboration with scientist ranging from physics and engineering to biomedical and clinical research, and 
		    with the ultimate goal of translating the research into industrial products to make the technologies 
		    available for the scientific community and hospital labs.</p> 
            </div>
	    <div class=" col-sm-5  col-md-5 col-lg-5" align="right">
	      <a class="twitter-timeline" data-width="400" data-height="400" data-dnt="true" data-theme="dark" href="https://twitter.com/MatsNilssonLab?ref_src=twsrc%5Etfw">Tweets by MatsNilssonLab</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>	
	    </div> 
	  </div> 

	      
<!-- Lab News -->		

      <h2>Lab News:</h2>
        <p>&nbsp;</p>
      <ul class="posts">
	
	<li>
          <span class="post-date">Nov. 25, 2019</span>
          <span class="post-link">NEWS: Xiaoyan Qian from the lab has successfully defended her PhD at Stockholm University<span>
        </li>
        <div class="well" style="background-color: #ebf0f5">
          <li class="media">
	      <div class="media-body">
           <strong class="media-heading"><a href="http://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1359663&dswid=-7721">Towards comprehensive cellular atlases: High-throughput cell mapping by in situ sequencing</a></strong>
            </div>
          </li>
        </div>
		  
<p>&nbsp;</p>      
	      
	<li>
          <span class="post-date">Nov. 18, 2019</span>
		<span class="post-link">Original article in <u>Nature Methods</u> on probabilistic cell typing using in situ sequencing</span>
        </li>
        <div class="well" style="background-color: #ebf0f5">
          <li class="media">
            <div class="media-body">
                <strong class="media-heading"><a href="https://www.nature.com/articles/s41592-019-0631-4">Probabilistic cell typing enables fine mapping of closely related cell types in situ</a></strong>
                <p><small>Qian X et al.</small></p>
		<div id="featureimage">
		  <img src="img/pciseq.png"; align="middle"; width="600px";>
		</div>
                <p>Understanding the function of a tissue requires knowing the spatial organization of its constituent cell types. 
			In the cerebral cortex, single-cell RNA sequencing (scRNA-seq) has revealed the genome-wide expression patterns 
			that define its many, closely related neuronal types, but cannot reveal their spatial arrangement. Here we 
			introduce probabilistic cell typing by in situ sequencing (pciSeq), an approach that leverages previous 
			scRNA-seq classification to identify cell types using multiplexed in situ RNA detection. We applied this method 
			by mapping the inhibitory neurons of mouse hippocampal area CA1, for which ground truth is available from 
			extensive previous work identifying their laminar organization. Our method identified these neuronal classes in 
			a spatial arrangement matching ground truth, and further identified multiple classes of isocortical pyramidal 
			cell in a pattern matching their known organization. This method will allow identifying the spatial organization 
			of closely related cell types across the brain and other tissues.
			<br><br>
			<strong class="media-heading"><a href="https://www.scilifelab.se/news/new-technique-enables-fine-mapping-of-closely-related-cells-in-situ/">SciLife Lab press release</a></strong>      
	      
                </p>   
              </div>
            </li>
          </div>
	
<p>&nbsp;</p>	
		  
	<li>
          <span class="post-date">Oct. 1, 2019</span>
		<span class="post-link">Original article in <u>EBioMedicine</u> on generation of OncoMaps with in situ sequencing in breast cancer</span>
        </li>
        <div class="well" style="background-color: #ebf0f5">
          <li class="media">
            <div class="media-body">
              <!--<img class="pull-left" style="margin-right:10px;" src="/img/csf.png" width="300px";> -->
                <strong class="media-heading"><a href="https://www.sciencedirect.com/science/article/pii/S2352396419306073">Generation of in situ sequencing based OncoMaps to spatially resolve gene expression profiles of diagnostic and prognostic markers in breast cancer</a></strong>
                <p><small>Svedlund J et al.</small></p>
		<div id="featureimage">
		  <img src="img/oncomap.jpg"; align="middle"; width="600px";>
		</div>
                <p> Our in situ approach spatially revealed intratumoral heterogeneity with regard to tumor subtype as well as to 
			the OncotypeDX recurrence score and even uncovered areas of minor cellular subpopulations. Since ISS-resolved 
			molecular profiles are linked to their histological context, a deeper analysis of the core and periphery of 
			tumor foci enabled identification of specific gene expression patterns associated with these morphologically 
			relevant regions.
                </p>   
              </div>
            </li>
          </div>
		  
<p>&nbsp;</p>	
		  
	<li>
          <span class="post-date">Aug. 26, 2019</span>
		<span class="post-link">Original article in <u>Nature Communications</u> on profiling surface proteins on exosomes</span>
        </li>
        <div class="well" style="background-color: #ebf0f5">
          <li class="media">
            <div class="media-body">
              <img class="pull-left" style="margin-right:10px;" src="img/exo.png" width="300px";>
                <strong class="media-heading"><a href="https://www.nature.com/articles/s41467-019-11486-1">Profiling surface proteins on individual exosomes using a proximity barcoding assay</a></strong>
                <p><small>Wu D et al.</small></p>
                Exosomes have been implicated in numerous biological processes, and they may serve as important disease markers. 
			Surface proteins on exosomes carry information about their tissues of origin. Because of the heterogeneity of 
			exosomes it is desirable to investigate them individually, but this has so far remained impractical. Here, we 
			demonstrate a proximity-dependent barcoding assay to profile surface proteins of individual exosomes using 
			antibody-DNA conjugates and next-generation sequencing. We first validate the method using artificial 
			streptavidin-oligonucleotide complexes, followed by analysis of the variable composition of surface proteins 
			on individual exosomes, derived from human body fluids or cell culture media. Exosomes from different sources 
			are characterized by the presence of specific combinations of surface proteins and their abundance, allowing 
			exosomes to be separately quantified in mixed samples to serve as markers for tissue-specific engagement in 
			disease.
                </p>   
              </div>
            </li>
          </div>
	      
<p>&nbsp;</p>	
	      
	<li>
          <span class="post-date">Jun. 7, 2019</span>
          <span class="post-link">Collaborative article with the group of Igor Adameyko investigating cell fate decisions in murine neural crest cells, now published in <u>Science</u></span>
        </li>
        <div class="well" style="background-color: #ebf0f5">
          <li class="media">
            <div class="media-body">
              <strong class="media-heading"><a href="https://science.sciencemag.org/content/364/6444/eaas9536.abstract">Spatiotemporal structure of cell fate decisions in murine neural crest</a></strong>
              <p><small>Soldatov R et al.</small></p>
            </div>
          </li>
        </div>
	      
<p>&nbsp;</p>	
	      
	<li>
          <span class="post-date">Apr. 23, 2019</span>
          <span class="post-link">Collaborative article with the group of Martin E Rottenberg in <u>Nature Communications</u>, using in situ sequencing to explore diversity in tuberculosis granuloma </span>
        </li>
        <div class="well" style="background-color: #ebf0f5">
          <li class="media">
            <div class="media-body">
              <strong class="media-heading"><a href="https://www.nature.com/articles/s41467-019-09816-4">Spatial and temporal localization of immune transcripts defines hallmarks and diversity in the tuberculosis granuloma</a></strong>
              <p><small>Carow B et al.</small></p>
            </div>
          </li>
        </div>
	      
<p>&nbsp;</p>	    
	    
	<li>
          <span class="post-date">May. 15, 2019</span>
          <span class="post-link">NEWS: Sibel Ciftci from the lab has successfully defended her PhD at Stockholm University<span>
        </li>
        <div class="well" style="background-color: #ebf0f5">
          <li class="media">
	      <div class="media-body">
           <strong class="media-heading"><a href="http://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1299600&dswid=-8601">Padlock Probe-Based Nucleic Acid Amplification Tests: Point-of-care Diagnostics of Infectious Diseases</a></strong>
            </div>
          </li>
        </div>
		  
<p>&nbsp;</p>
			  
	<li>
          <span class="post-date">Dec 6, 2018</span>
          <span class="post-link">NEWS: Chan Zuckerberg Initiative award for Neurodegeneration Challenge Network to profile and define Parkinson's disease<span>
        </li>
        <div class="well" style="background-color: #ebf0f5">
          <li class="media">
            <div class="media-body">
              <div id="featureimage">
                <img src="img/czi.png"; align="middle"; width="800px";>
              </div>
              <p></p>
	
              <p>The group of Ernest Arenas has been awarded a grant from CZI to systematically analyze and provide an unbiased understanding of the 
		      cell types and molecular mechanisms involved in Parkinson's disease.
		      The project will be performed in collaboration with Per Svenningsson (KI), Sten Linnarsson (KI), and Mats Nilsson (SU).
	      <br>
	      <br>
	        <strong class="media-heading"><a href="https://go.chanzuckerberg.com/NCN#!">More information: CZI Neurodegeneration Challenge Network</a></strong>
              <br>
	      <br>
		<strong class="media-heading"><a href="https://www.scilifelab.se/news/parkinsons-CZI/">SciLife Lab press release</a></strong>      
	      </p>
            </div>
          </li>
        </div>
           

	  <p>&nbsp;</p>

<!-- End news section -->
	<hr>
<!-- Start 'About the lab' section -->
      </ul>
      <div class="news">	      
        <h2>About the Department and SciLifeLab:</h2>
	  <ul class="posts">	
		<p>The Department is mainly located with the other Departments of Chemistry and Life Sciences in the Arrhenius 
		    Laboratories for Natural Sciences, which are situated in the northern part of the University Campus at 
		    Frescati. Presently more than 300 people are working at the Department of which about 110 are PhD students 
		    engaged in internationally highly recognized research covering a broad range of subjects. The Department is 
		    also deeply involved in teaching, with courses at all undergraduate levels, including a wide range of 
		    Master courses. A close link between the undergraduate program and the research projects has since long 
		    been a tradition and a trademark of the Department. Science for Life Laboratory, where many of our 
		    researchers are based, is also closely linked to the department.</p>
		  <p>SciLifeLab is a Swedish national center for molecular biosciences with focus on health and environmental 
			  research. The center combines frontline technical expertise with advanced knowledge of translational 
			  medicine and molecular bioscience. SciLifeLab is hosted by four Swedish universities (Karolinska 
			  Institutet, KTH Royal Institute of Technology, Stockholm University and Uppsala University) and 
			  collaborates with several other universities.
		<br>
		SciLifeLab was established in 2010 and appointed a national center in 2013 by the Swedish government. 
			  More than 100 research groups are associated with SciLifeLab, which is situated at two nodes in 
			  Stockholm and Uppsala.</p>
            
	  </ul>
        </div>
		
<p>&nbsp;</p>
		
<!-- Start 'Contact' section -->	
		
    <hr>

      <div class="news">
        <p>&nbsp;</p>
        <h2>Contact Information</h2>
	<ul class="posts">
	  <div class="row">
	    <div class=" col-sm-4  col-md-4 col-lg-4" align="left">  
              <h4>Visiting address:</h4>
                <p>Science for Life Laboratory<br>
                   Gamma 4<br>
		   Tomtebodavägen 23A<br>
                   171 65 Solna, Sweden</p>
                   <a href="https://www.google.com/maps/place/Science+For+Life+Laboratory/@59.3502341,18.0230999,19.43z/data=!4m5!3m4!1s0x465f9d9c044c1fe5:0xfff6f935131557c1!8m2!3d59.3501961!4d18.0227945" target="_blank"> <img style='width: 30%; object-fit: contain' src="img/mapsicon.jpg" border="0" alt=""> </a> 
            </div>
	    <div class=" col-sm-4  col-md-4 col-lg-4" align="left">
	      <h4>Mailing address:</h4>
                <p>Science for Life Laboratory<br>
                   Box 1031<br>
                   171 21 Solna, Sweden</p>	
	    </div> 
	    <div class=" col-sm-4  col-md-4 col-lg-4" align="left">
	      <h4>For deliveries:</h4>
                <p>Science for Life Laboratory<br>
                   Gamma 4<br>
		   Tomtebodavägen 23B<br>
                   171 65 Solna, Sweden</p>	
	    </div>
          </div>
	</ul>
      </div> 
	

		
<!-- End 'Contact' section -->		
      	
    </div>
   </div>
  </div>
  <p>&nbsp;</p>
  
<!-- Footer section -->	
	
  <footer class="footer">
    <div class="wrapfoot">
      <div class="row">
	<div class=" col-sm-4  col-md-3 col-lg-3">
          <ul>
            <li><font color="white">Mats Nilsson Lab</font></li>
	    <li><a href="https://twitter.com/MatsNilssonLab" class="twitter-follow-button" data-show-count="false">Follow @MatsNilssonLab</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script></li>
          </ul>
        </div>
        <div class=" col-sm-8  col-md-9 col-lg-9" align="right">
	  <ul>
            <li><a href="https://www.scilifelab.se/researchers/mats-nilsson/"><font color=#ccd5df>Mats Nilsson SciLifeLab</font></a></li>
            <li><a href="https://www.scilifelab.se/"><font color=#ccd5df>SciLifeLab</font></a></li>
	    <li><a href="https://www.dbb.su.se/"><font color=#ccd5df>Department of Biochemistry and Biophysics</font></a></li>
            <li><a href="https://www.su.se/english/"><font color=#ccd5df>Stockholm University</font></a></li>
          </ul>   
        </div>   
      </div>
      <div class="row">
	<div class=" col-sm-4  col-md-4 col-lg-4">  
        </div>
        <div class=" col-sm-4  col-md-4 col-lg-4" align="center">
		<p><font color=#ccd5df>Updated: Dec 5, 2019</font></p>
	</div>   
        <div class=" col-sm-4  col-md-4 col-lg-4">
	</div>
      </div>
    </div>
  </footer>
	
<!-- End footer -->	

	



</body>
</html>

