<!DOCTYPE html>
<html>
<head>
  <title>Mats Nilsson Lab</title>
  <meta charset="utf-8"/>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
  <link href='https://fonts.googleapis.com/css?family=Roboto:300,400,700' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" type="text/css" href="./css/main.css">

</head>
<body>
  <header class="container">
	  <link rel="icon" href="img/moldia.png">

    <div class="navigation">
      <ul>
        <li><a class="page-link" href="">Home</a></li>
        <li><a class="page-link" href="research/">Research</a></li>
        <li><a class="page-link" href="publications/">Publications</a></li>
        <li><a class="page-link" href="people/">People</a></li>
        <li><a class="page-link" href="links/">Links</a></li>
      </ul>
    </div>
  </header>
  
	<p>&nbsp;</p>
	
	
<!-- LARGE TITLE BANNER -->	
	
  <section class="jumbotron">
    <div class="container">
      <div class="row text-center">
        <h1>UNDER CONSTRUCTION</h1>
        <p>&nbsp;</p>
        <h3>Stockholm University</h3>	      
        <h4>Department of Biochemistry and Biophysics</h4>
        <h4>SciLifeLab</h4>
      </div>
    </div>
  </section>
  
  <p>&nbsp;</p>
	
<!-- News post section -->	
  
  <div class="wrap">
    <div class="post">
      <div class="news">
        
<!-- Pinned headline -->
	      
<!--	
	<h2>Title</h2>
	  <ul class="posts">
            <li>
              <span class="post-link"><p><a href="___"</a></p></span>
	    </li>
	  </ul>
	      
        <p>&nbsp;</p>
-->	      
	    
<!-- End pinned headline -->	      
	      
        <h2>Lab News:</h2>
        <p>&nbsp;</p>
        <ul class="posts">
          
<!-- TO ADD NEWS ITEM, COPY AND PASTE TEMPLATE BELOW:	  

          <li>
            <span class="post-date">TYPE DATE HERE</span>
            <span class="post-link">POST TITLE HERE</span>
          </li>
          <div class="well">
            <li class="media">
              <div class="media-body">
                <img class="pull-left" style="margin-right:10px;" src="IMAGE FILE HERE" height="300px";>
                  <strong class="media-heading"><a href="LINK TO ARTICLE/NEWS HERE">TITLE FOR LINK HERE</a></strong>
                <p><small>SHORTHAND AUTHORS HERE</small></p>
                 <p>PARAGRAPH TEXT HERE</p>
                FOR ADDITIONAL LINKS INCLUDE LINES BELOW, OTHERWISE REMOVE
                <p><a href="LINK HERE">TITLE FOR LINK</a></p>
                <p><a href="LINK HERE">TITLE FOR LINK</a></p>
              </div>
             </li>
           </div>
           <p>&nbsp;</p>

:END TEMPLATE -->
		
	<li>
          <span class="post-date">Nov. 18, 2019</span>
		<span class="post-link">Original article in <u>Nature Methods</u> on probabilistic cell typing using in situ sequencing</span>
        </li>
        <div class="well">
          <li class="media">
            <div class="media-body">
              <!--<img class="pull-left" style="margin-right:10px;" src="/img/csf.png" width="300px";> -->
                <strong class="media-heading"><a href="https://www.nature.com/articles/s41592-019-0631-4">Probabilistic cell typing enables fine mapping of closely related cell types in situ</a></strong>
                <p><small>Qian X et al.</small></p>
		<div id="featureimage">
		  <img src="img/pciseq.png"; align="middle"; width="600px";>
		</div>
                <p>Understanding the function of a tissue requires knowing the spatial organization of its constituent cell types. 
			In the cerebral cortex, single-cell RNA sequencing (scRNA-seq) has revealed the genome-wide expression patterns 
			that define its many, closely related neuronal types, but cannot reveal their spatial arrangement. Here we 
			introduce probabilistic cell typing by in situ sequencing (pciSeq), an approach that leverages previous 
			scRNA-seq classification to identify cell types using multiplexed in situ RNA detection. We applied this method 
			by mapping the inhibitory neurons of mouse hippocampal area CA1, for which ground truth is available from 
			extensive previous work identifying their laminar organization. Our method identified these neuronal classes in 
			a spatial arrangement matching ground truth, and further identified multiple classes of isocortical pyramidal 
			cell in a pattern matching their known organization. This method will allow identifying the spatial organization 
			of closely related cell types across the brain and other tissues.
                </p>   
              </div>
            </li>
          </div>
		
<p>&nbsp;</p>
		
     <li>
          <span class="post-date">Date</span>
          <span class="post-link">NEWS: title<span>
        </li>
        <div class="well">
          <li class="media">
	      <div class="media-body">
           
            </div>
          </li>
        </div>
           
<p>&nbsp;</p>
		
		
	<li>
          <span class="post-date">Date</span>
          <span class="post-link">Collaborative article</span>
        </li>
        <div class="well">
          <li class="media">
            <div class="media-body">
              <strong class="media-heading"><a href="">Title</a></strong>
              <p><small>author</small></p>
            </div>
          </li>
        </div>
	<p>&nbsp;</p>
		
	<li>
          <span class="post-date">Date</span>
	  <span class="post-link">Title</span>
        </li>
        <div class="well">
          <li class="media">
            <div class="media-body">
              <img class="pull-left" style="margin-right:10px;" src="/img/EC-jbc.jpg" width="300px";>
                <strong class="media-heading"><a href="">Link title</strong>
                <p><small>author</small></p>Abstract
                </p>   
              </div>
            </li>
          </div>
		  
<p>&nbsp;</p>	
		
	  
	<li>
          <span class="post-date">Dec 6, 2018</span>
          <span class="post-link">NEWS: Chan Zuckerberg Initiative award for Neurodegeneration Challenge Network to profile and define Parkinson's disease<span>
        </li>
        <div class="well">
          <li class="media">
            <div class="media-body">
              <div id="featureimage">
                <img src="img/czi.png"; align="middle"; width="600px";>
              </div>
              <p></p>
	
              <p>The group of Ernest Arenas has been awarded a grant from CZI to systematically analyze and provide an unbiased understanding of the 
		      cell types and molecular mechanisms involved in Parkinson's disease.
		      The project will be performed in collaboration with Per Svenningsson (KI), Sten Linnarsson (KI), and Mats Nilsson (SU).
	      <br>
	      <br>
	        <strong class="media-heading"><a href="https://go.chanzuckerberg.com/NCN#!">More information: CZI Neurodegeneration Challenge Network</a></strong>
              <br>
	      <br>
		<strong class="media-heading"><a href="https://www.scilifelab.se/news/parkinsons-CZI/">SciLife Lab press release</a></strong>      
	      </p>
            </div>
          </li>
        </div>
           

	  <p>&nbsp;</p>

<!-- End news section -->

<!-- Start 'About the lab' section -->
      </ul>
      <div class="news">	      
        <h2>About the lab:</h2>
	  <ul class="posts">	
            <p>Our work is focused on development of novel molecular analysis concept for use in research and diagnostics, 
		    with primary focus on infectious and cancer diagnostics. We address development of both fundamental assay 
		    architecture and novel devices. Our research is based on a cross-disciplinary approach involving extensive 
		    collaboration with scientist ranging from physics and engineering to biomedical and clinical research, and 
		    with the ultimate goal of translating the research into industrial products to make the technologies 
		    available for the scientific community and hospital labs.

		    A current trend in drug development is that novel therapies are more targeted to a certain molecular 
		    defect. Thus, molecular diagnostics has become a central element to be able to prescribe the right drug. 
		    Powerful research tools are not always suitable for the diagnostic setting, where tests needs to be very 
		    reliable, automated, usually relatively rapid, inexpensive, and fit the sample logistics and throughput of 
		    a typical hospital lab.

		    The fundament of our research is based on advanced molecular tools employing nucleic acid processing 
		    enzymes and DNA circularization reactions, as used in the padlock and selector probe technologies developed 
		    in our lab. The selector technology enables highly multiplexed targeted ultra-deep next generation 
		    sequencing suitable for diagnostics and is now commercially available as tha HaloPlex assay. Padlock probes 
		    combined with RCA provides interesting analytical advantages. First, due to the single-molecule sensitivity 
		    of these assays, they can be used for highly precise digital quantification. Second, they can be used to 
		    elicit novel magnetic or electric biosensor readouts, that can be used for hand held devices. Finally, they 
		    can be implemented in situ to detect and digitally quantify DNA and RNA sequences resolving 
		    single-nucleotide variants at micro-meter resolution, which we utilize in our in situ sequencing approach, 
		    applying next generation sequencing in situ. With this technique we can sequence DNA and RNA molecules in 
		    the preserved context of fixed cells and tissue sections and it can be applied for massively multiplexed 
		    expression profiling, splice variant mapping, and mutation detection in situ.</p>
            <p>The Department is mainly located with the other Departments of Chemistry and Life Sciences in the Arrhenius 
		    Laboratories for Natural Sciences, which are situated in the northern part of the University Campus at 
		    Frescati. Presently more than 300 people are working at the Department of which about 110 are PhD students 
		    engaged in internationally highly recognized research covering a broad range of subjects. The Department is 
		    also deeply involved in teaching, with courses at all undergraduate levels, including a wide range of 
		    Master courses. A close link between the undergraduate program and the research projects has since long 
		    been a tradition and a trademark of the Department. Science for Life Laboratory, where many of our 
		    researchers are based, is also closely linked to the department.</p>
		  <p>SciLifeLab, Science for Life Laboratory, is an institution for the advancement of molecular biosciences in 
		    Sweden. We are funded as a national research infrastructure by the Swedish government. Our organization 
		    leverages the unique strengths of individual researchers across Sweden into a focused resource for the life 
		    science community. We provide access for thousands of researchers to the cutting-edge instrumentation and 
		    deep scientific expertise necessary to be internationally competitive in bioscience research. This 
		    infrastructure is supported and developed by our research community, including internationally recognized 
		    experts in life science and technology. Our facilities and expertise create a unique environment for 
		    carrying out health and environmental research at the highest level.

		SciLifeLab started out in 2010 as a joint effort between four universities: Karolinska Institutet, KTH Royal 
		    Institute of Technology, Stockholm University and Uppsala University. Today, we support research activities 
		    at all major Swedish universities.</p>
            
	  </ul>
        </div>
<p>&nbsp;</p>
<!-- Start 'Contact' section -->	      
    <hr>

      <div class="news">
        <p>&nbsp;</p>
        <h2>Contact Information</h2>
	<ul class="posts">
	  <div class="row">
	    <div class=" col-sm-4  col-md-4 col-lg-4" align="left">  
              <h4>Visiting address:</h4>
                <p>Science for Life Laboratory<br>
                   Gamma 4<br>
		   Tomtebodavägen 23A<br>
                   171 65 Solna, Sweden</p>
                   <a href="https://www.google.com/maps/place/Solnav%C3%A4gen+9,+171+65+Solna/@59.3497337,18.0233539,17z/data=!3m1!4b1!4m5!3m4!1s0x465f9d9c18ba64f7:0xd8d92683a7bdfd6d!8m2!3d59.3497337!4d18.0255426" target="_blank"> <img style='width: 30%; object-fit: contain' src="img/mapsicon.jpg" border="0" alt=""> </a> 
            </div>
	    <div class=" col-sm-4  col-md-4 col-lg-4" align="left">
	      <h4>Mailing address:</h4>
                <p>Science for Life Laboratory<br>
                   Box 1031<br>
                   171 21 Solna, Sweden</p>	
	    </div> 
	    <div class=" col-sm-4  col-md-4 col-lg-4" align="left">
	      <h4>For deliveries:</h4>
                <p>Science for Life Laboratory<br>
                   Gamma 4<br>
		   Tomtebodavägen 23B<br>
                   171 65 Solna, Sweden</p>	
	    </div>
            
          </div>
	</ul>
      </div> 
	

		
<!-- End 'Contact' section -->		
      	
    </div>
   </div>
  </div>
  <p>&nbsp;</p>
  


<!-- Footer section -->	
	
  <footer class="footer">
    <div class="wrapfoot">
      <div class="row">
	<div class=" col-sm-4  col-md-3 col-lg-3">
          <ul>
            <li>Mats Nilsson Lab</li>
            <li><a href="mailto:(email)">email(@)scilife</a></li>
          </ul>
        </div>
        <div class=" col-sm-8  col-md-9 col-lg-9" align="right">
	  <ul>
            <li><a href="https://www.scilifelab.se/researchers/mats-nilsson/">Mats Nilsson SciLifeLab</a></li>
            <li><a href="https://www.dbb.su.se/">Department of Biochemistry and Biophysics</a></li>
            <li><a href="https://www.scilifelab.se/">SciLifeLab</a></li>
	    <li><a href="https://www.su.se/english/">Stockholm University</a></li>
          </ul>   
        </div>   
      </div>
      <div class="row">
	<div class=" col-sm-4  col-md-4 col-lg-4">  
        </div>
        <div class=" col-sm-4  col-md-4 col-lg-4" align="center">
		<p>Updated: Nov 11, 2018</p>
	</div>   
        <div class=" col-sm-4  col-md-4 col-lg-4">
	</div>
      </div>
    </div>
  </footer>
	
<!-- End footer -->	

	



</body>
</html>

